### PR TITLE
Berlin Open Data portal

### PR DESCRIPTION
**Pull request by knudmoeller**
text, metadata and images for daten.berlin.de aka "Berlin Open Data"

### CODE REVIEW

**Comment by dnsos** [comment]
Hey @knudmoeller, thank you for the contribution. We are happy to include the data portal projects in our list.

The `repository` field in the front matter actually doesn't expect a non-repository link (and this might cause issues with our script for fetching data from the GitHub API). However, we see that it makes sense to present the repositories as a bundle. I will come back to you here with a solution very soon.

**Comment by knudmoeller** [comment]
Cool, thanks! Let me know if you need any further input from me. If the non-repository link doesn't work out, I could create a dummy meta-repository "Berlin Open Data" with just a README and links to the individual 'real' repos. I'm not sure what would be better.

File: src/projects/berlin-open-data.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+layout: layouts/project.liquid
+tags: ["Open Data"]
+id: berlin-open-data
+title: Berlin Open Data
+abstract: Offene Daten der Berliner Verwaltungen und landeseigenen Betriebe
+languages: ["Python", "PHP", "JavaScript"]
+license: GPL-2.0, AGPL-3.0, MIT (verschiedene Repositories)
+link: https://daten.berlin.de
+repository: https://github.com/search?q=org%3Aberlinonline+topic%3Aberlinopendata
```
Comment (dnsos) [inline src/projects/berlin-open-data.md:10]: We now have a field specifically for this case: `repository_collection`. Simply rename the key here.

**Review by dnsos — CHANGES_REQUESTED** [review]
So, we have decided to provide the option for repository collections generally and now added a front matter field for this in #153 

If you could simply update the name of your field as noted in my comment, we're good to go and merge this. Thank you! Glad the repos are coming to the site.

**Comment by knudmoeller** [comment]
Ok, I've made the change in the PR branch. That's all you need, right?

**Review by dnsos — APPROVED** [review]

**Comment by dnsos** [comment]
Perfect, thank you. Yes, that's all that's needed. Merging this now!

**Pull request merged**